### ISSUE TITLE
Most Fizzlers need to be set to handle 8 Positions

### ISSUE THREAD

**Issue by ellingtonvengaboysmuseum** [BEEmod/BEE2-items#2332]
Currently Most custom Fizzlers are set to handle 6 Positions

they need to be set to handle 8 Positions

**Comment by Endermage77** [comment]
I don't think there's such a thing as Handle-6.

**Comment by ellingtonvengaboysmuseum** [comment]
handle 6 positions is what their set to in the vbsp config...

**Comment by Luke18033** [comment]
> **BEE2-items Wiki** says:
`HANDLE_6_POSITIONS`: Allows centering in two orientations, and offset on the 4 sides. The instance should be centered, with the sides in the `y` axis. When offset, the `+x` side faces toward the center (and won't extend into other voxels).

**Comment by Luke18033** [comment]
The problem with this is that it would make it so you couldn't place fizzlers in the center of a voxel. I think at some point somebody suggested making a stylevar for this, and @TeamSpen210 even pointed out that they'd keep their current position when the handle was swapped. I can't find an issue for it, so it was probably suggested on Discord or Gitter.

**Comment by LautaroL20** [comment]
Yep, i tested this and works like a charm. This stylevar should definitelly be implemented.

**Comment by WhoozYerBuddy** [comment]
Light bridges too.

**Comment by LautaroL20** [comment]
Title updated.

**Comment by Luke18033** [comment]
This would be more of an application issue I think, we'd want to swap all 6-way items to 8-way automatically.
